### PR TITLE
Improve the behaviour of `½`

### PR DESCRIPTION
**Pull request by Lyxal**
Goes towards closing #1164

### CODE REVIEW

File: vyxal/elements.py [hunk 1]
```diff
@@ -1962,7 +1962,9 @@ def halve(lhs, ctx):
     ts = vy_type(lhs)
     return {
         NUMBER_TYPE: lambda: sympy.Rational(lhs, 2),
-        str: lambda: wrap(lhs, math.ceil(len(lhs) / 2), ctx=ctx),
+        str: lambda: wrap(lhs, math.ceil(len(lhs) / 2), ctx=ctx)
+        if len(lhs) > 1
+        else [lhs, ""],
     }.get(ts, lambda: vectorise(halve, lhs, ctx=ctx))()
```
Comment (cgccuser) [inline vyxal/elements.py:1968]: Do you want to make this whole thing just `return divide(lhs, 2)` lol

Comment (Lyxal) [reply vyxal/elements.py:1968]: No, because it wouldn't work lol

**Review by Steffan153 — APPROVED** [review]

**Comment by cgccuser** [comment]
I have to say, it feels weird that `½` isn't the same as `2 /`. We may want to rethink the behavior of `/`. I guess it can't be changed now because it'd be breaking, but we should take a look at it again for v3.

**Review by chunkybanana — APPROVED** [review]
`/` is still sus tho

**Pull request merged**